server/controllers/transactions.js

```diff
@@ -1,26 +1,59 @@
-import models from '../models';
+import models, { sequelize } from '../models';
 
 /**
  * Get a transaction
  */
-export const getOne = (req, res) => {
+export const getOne = (req, res, next) => {
   Promise.all([
-    models.User.findOne({
-      where: { CollectiveId: req.transaction.HostCollectiveId },
-      include: [ { model: models.Collective, as: 'collective' } ]
+    models.Collective.findOne({
+      where: { id: req.transaction.HostCollectiveId },
     }),
     req.transaction.getCollective(),
     req.transaction.getFromCollective(),
-    req.transaction.getCreatedByUser()
+    req.transaction.getCreatedByUser(),
+    
+    // TODO: we shouldn't need two paths to fetch a host
+    // User->Collective relationship needs to be stored in the Members table
```

Comment:
We don't need a relationship for that 1:1.
Instead, `Users` table should be eventually moved to `ConnectedAccounts` and use the `ConnectedAccounts.CollectiveId` to references the Collective it is associated to.

Likewise, we should introduce a new table `Locations` that should also references a `CollectiveId`. Location could be of type "residence", "billing", "shipping", "venue" (for events).

Reply:
Agreed it should be moved - eventually. That eventually isn't going to be in next few months. It's a big migration and we are already behind on our goals for the quarter. And in the meantime, everywhere we need to fetch a User and it's Collective, requires two different pathways (one for USER and one for ORG). Creates more instability. Fixed two such pathways yesterday alone.

yep, addresses need to be separate table as well. That's easier to do.